ENG-5258: Customizing workbench images with Elyra (#833)
opendatahub-io/opendatahub-documentation: modules/creating-a-custom-image-from-default-image.adoc

.Prerequisites

* You know which default image you want to use as the base for your custom image. 
+
[IMPORTANT]
====

> If you want to build a custom Elyra image, you must customize an existing {productname-short} Elyra image.

Reviewer: Im thinking more about highlighting that you need to edit it directly. What you think of this? 
```suggestion
If you want to customize an Elyra image, you must modify an existing {productname-short} Elyra image.  
```

> If you want to customize an Elyra image, the base image that you copy must be an {productname-short} Elyra image.

Reviewer 2: ```suggestion
If you want to create a custom Elyra image, you must specify an {productname-short} Elyra image as the base image.
```
